pkg/sysinfo/cgroup2_linux.go

```diff
@@ -5,9 +5,9 @@ import (
 	"path"
 	"strings"
 
+	"github.com/containerd/cgroups"
 	cgroupsV2 "github.com/containerd/cgroups/v2"
 	"github.com/containerd/containerd/pkg/userns"
-	"github.com/opencontainers/runc/libcontainer/cgroups"
 	"github.com/sirupsen/logrus"
 )
```

Comment:
This caused a regression: now `docker info` always shows `WARNING: No swap limit support`.

Because `containerd/cgroups.ParseCgroupFile` (purposely) does not support the unified hierarchy: 
- https://github.com/containerd/cgroups/pull/232

Reply:
Ah, dang that's what I initially was afraid of https://github.com/moby/moby/pull/43347#discussion_r822789642